benchmarks/configs/names.py

```diff
@@ -0,0 +1,83 @@
+# Copyright 2025 Thousand Brains Project
+#
+# Copyright may exist in Contributors' modifications
+# and/or contributions to the work.
+#
+# Use of this source code is governed by the MIT
+# license that can be found in the LICENSE file or at
+# https://opensource.org/licenses/MIT.
+
+from dataclasses import dataclass, fields
+
```

Comment:
Maybe worth adding some documentation here for why we have this approach using dataclasses ("Prevent drift in names etc.")? Just imaging someone coming back to this in 6 months might be a bit confused by this decision.

Reply:
Added, now reads:
```python
"""The names of declared experiments grouped by category.

This module keeps experiment names separate from the configuration for the
experiments. The reason for doing this is so that we can import the configurations
selectively to avoid importing uninstalled dependencies (e.g., not installing a
specific simulator).

The use of dataclasses assists in raising early errors when experiment names defined
here and the corresponding experiment configurations drift apart. For additional
discussion, see: https://github.com/thousandbrainsproject/tbp.monty/pull/153.
"""
```